backend/src/main/java/ecse428/backend/dto/MealDto.java

```diff
@@ -0,0 +1,97 @@
+package ecse428.backend.dto;
+
+import ecse428.backend.model.Meal;
+import ecse428.backend.model.SmartEats.Ingredient;
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Size;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+public class MealDto {
+
```

Comment:
I don't know how the model is represented but It would be nice to have the dietary restrictions for this meal, so that we can filter them later (Get by restriction or something)

Reply:
do you mean have a dietary restrictions field for each meal ?

Reply:
Yes, a set of dietary restrictions, just like each customer

Reply:
ahh okk i see what you mean. In this case, wouldn't it be better to cross match the list of ingredients of each meal with the list of dietary restrictions. Then we can generate a list of dietary restrictions for each meal dynamically. Otherwise it just seems like we would be storing duplicate info on the db ?

Reply:
@KRouhana this might be an extra feature, we could decide to add it or not. The simplest way for now would be to add the list of restrictions manually each time.